planet/cli/orders.py

```diff
@@ -253,11 +253,26 @@ async def create(ctx, request: str, pretty):
               default=False,
               is_flag=True,
               help='Send email notification when order is complete.')
+@click.option('--archive-type',
+              type=click.Choice(['zip']),
+              help="Optionally zip archive each item bundle.")
+@click.option('--archive-filename',
+              default='{{name}}-{{order_id}}.zip',
+              show_default=True,
+              help="Templated filename for archived bundles or orders.")
+@click.option('--single-archive',
+              is_flag=True,
+              default=False,
+              show_default=True,
+              help="Optionally zip archive all item bundles together.")
```

Comment:
This is a great addition, thanks for including it.

Reply:
Yes indeed, this was an oversight. It's easy to imagine users only using, for example, `--archive-type zip` and no cloud config at all.